### DISCUSSION TITLE
［委托］高价收取艾莲小姐的贴身衣物喵

### DISCUSSION THREAD

**Discussion by abyssal-sea-c** [share121/inter-knot#5448] [委托]
高价回收喵
有意者请联系:0721onani@0721.yuuu喵
<img width="508" alt="illust_120112974_20240908_233656" src="https://github.com/user-attachments/assets/5a7e38a3-ba22-4eaf-87c8-c0bf6f91af74">

**Comment by obyz-wzr** [comment]
我比他多出一元

**Comment by zhongsanZN** [comment]
我比上楼多出两元

**Comment by KELERSAMA** [comment]
我比楼上多出三元
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by fzdqz** [comment]
我比楼上多出四块

**Comment by keyuanya** [comment]
我比楼上多出十块
